feature_annotations.yaml

```diff
@@ -0,0 +1,28 @@
+# This code-annotations configuration file supports OEP-17, Feature Toggles.
```

Comment:
Nit: Add a reference to the OEP: https://open-edx-proposals.readthedocs.io/en/latest/oep-0017-bp-feature-toggles.html